test/second_q/problems/resources/H2_631g_ferm_op_active_space.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
TODO: remove the non-json files in this folder, which should no longer be needed now